builds/misc/templates/build-packages.yaml

```diff
@@ -142,4 +142,75 @@ stages:
             inputs:
               PathtoPublish: '$(build.artifactstagingdirectory)'
               ArtifactName: 'iotedged-$(os)-$(arch)'
-            condition: or(eq(${{ parameters['E2EBuild'] }}, false), eq(variables.arch,'amd64'))
\ No newline at end of file
+            condition: or(eq(${{ parameters['E2EBuild'] }}, false), eq(variables.arch,'amd64'))
+
+      ################################################################################
+      - job: mariner_linux
+      ################################################################################
+        displayName: Mariner_Linux
+        pool:
+          vmImage: 'ubuntu-18.04'
+        variables:
+          NugetSecurityAnalysisWarningLevel: warn
+        strategy:
+          matrix:
+            CBL-Mariner1.0-amd64:
+              arch: amd64
+              os: mariner
+              release: tags/1.0-stable
+              target.iotedged: builds/mariner/out/RPMS
+        steps:
+          - bash: |
+              BASE_VERSION=`cat $BUILD_SOURCESDIRECTORY/edgelet/version.txt`
+              VERSION="$BASE_VERSION$BUILD_BUILDNUMBER"
+              echo "##vso[task.setvariable variable=VERSION;]$VERSION"
+              echo "##vso[task.setvariable variable=PACKAGE_ARCH;]$(arch)"
+              echo "PACKAGE_OS=$(os)"
+              echo "##vso[task.setvariable variable=MARINER_RELEASE;]$(release)"
+              mariner_arch=$(arch)
+              if [ $mariner_arch == "amd64" ]; then
+                mariner_arch="x86_64"
+              fi
+              echo "##vso[task.setvariable variable=MARINER_ARCH;]$mariner_arch"
+            displayName: Set Version
+          - task: AzureKeyVault@1
+            displayName: 'Azure Key Vault: EdgeBuildkv'
+            inputs:
+              azureSubscription: $(az.subscription)
+              KeyVaultName: 'edgebuildkv'
+              SecretsFilter: >- 
+                IotEdge1-GitHub-PAT
+          - task: PowerShell@2
+            displayName: 'Download aziot-identity-service'
+            inputs:
+              filePath: scripts/local/test/DownloadIdentityService.ps1
+            env:
+              GITHUB_TOKEN: $(IotEdge1-GitHub-PAT)
+              ARTIFACT_NAME: 'packages_mariner_$(arch)'
+              PACKAGE_FILTER: 'aziot-identity-service-*.cm1.$(MARINER_ARCH).rpm'
+              DOWNLOAD_PATH: $(Build.StagingDirectory)
+              IDENTITY_SERVICE_COMMIT: 'f332cc521deea6072d64ed151fb283da340bb38a'
```

Comment:
Please make this env var configurable as part of the pipeline or preferably have it automatically be parsed from the  edgelet/cargo.lock file i.e. `source = "git+https://github.com/Azure/iot-identity-service?branch=release/1.2#8fc413a9910588b2949eca8ad1ea28246c066f08"`

Reply:
Right now the section IDENTITY_SERVICE_COMMIT: 'f332cc521deea6072d64ed151fb283da340bb38a' is fixed since there has not been an official release of IIS with the marine RPM yet. once there is we can remove that part since it will auto populate the commit.